capstone/src/main/webapp/script.js

```diff
@@ -203,3 +205,9 @@ function saveClubChanges() {
   alert('Changes submitted!');
 }
 
+async function getImageUrl(logoKey) {
```

Comment:
nit: Have you tried removing the other getImageUrl function in explore-loader.js and reusing this one for both JS files? I think it should work because index.html has both script.js and explore-loader.js loaded. I don't use async / await so I wasn't able to try it for the profile pictures :(
https://stackoverflow.com/questions/3809862/can-we-call-the-function-written-in-one-javascript-in-another-js-file/3811763#:~:text=As%20long%20as%20both%20are,in%20the%20same%20JS%20file.&text=If%20all%20files%20are%20included,that%20you%20write%20in%20one%20.

Reply:
Thanks!